client/www/_emails/markdown/nov2024.md

```diff
@@ -0,0 +1,60 @@
+![](https://paper-attachments.dropboxusercontent.com/s_DF8F10A9009F2A236BC7D07C4EC05DDA50E4FB82F40AA98593D3B98A1A7EA3DC_1730238267270_instant_header.png)
+
+
+Hey crew! It’s time for another edition of Instant News 🙂 Here’s a slice of what happened in November!
+
+**Announcing enforced types on the backend**
+
+![](https://paper-attachments.dropboxusercontent.com/s_B097BA2FA68FBDBEF71F30F95558A3D23729AED0A557430F53FE49A682293BBC_1733450241867_image.png)
```

Comment:
Is it ok to embed the dropbox paper images? I'm a little surprised they don't block external domains or something.

Reply:
These do work! The previous newsletter used dropbox paper images as well, and the images load on our [preview tool](https://www.instantdb.com/intern/emails/oct2024). Inspecting delivered emails in Postmark shows the images rendering correctly too.

Reasonable for us to self-host the images tho, will look an alternative solution!